providers/azure/README.md

```diff
@@ -273,6 +262,129 @@ To verify that virtual kubelet has started, run:
 ```cli
   kubectl --namespace=default get pods -l "app=virtual-kubelet-virtual-kubelet"
 ```
+##  Create an AKS cluster with VNet
+
+  Run the following commands to create an AKS cluster with a new Azure virtual network. Also, create two subnets. One will be delegated to the cluster and the other will be delegated to Azure Container Instances. 
+
+### Create an Azure virtual network and subnets 
+
+  First, set the following variables for your VNet range and two subnet ranges within that VNet. The following ranges are recommended for those just trying out the connector with VNet. 
+
+  **Bash**
+  ```cli
+    export VNET_RANGE=10.8.0.0/8  
+    export CLUSTER_SUBNET_RANGE=10.240.0.0/16 
+    export ACI_SUBNET_RANGE=10.241.0.0/16 
+    export VNET_NAME=myAKSVNet 
+    export CLUSTER_SUBNET_NAME=myAKSSubnet 
+    export ACI_SUBNET_NAME=myACISubnet 
+    export AKS_CLUSTER_RG=myresourcegroup 
+    export KUBE_DNS_IP=10.0.0.10
+  ```
+  Run the following command to create a virtual network within Azure, and a subnet within that VNet. The subnet will be dedicated to the nodes in the AKS cluster.
+
+    ```cli
+    az network vnet create \
+    --resource-group $AKS_CLUSTER_RG \
+    --name $VNET_NAME \
+    --address-prefixes $VNET_RANGE \
+    --subnet-name $CLUSTER_SUBNET_NAME \
+    --subnet-prefix $CLUSTER_SUBNET_RANGE
+    ```
+
+Create a subnet that will be delegated to just resources within ACI, note that this needs to be an empty subnet, but within the same VNet that you already created. 
+
+```cli
+az network vnet subnet create \
+    --resource-group $AKS_CLUSTER_RG \
+    --vnet-name $VNET_NAME \
+    --name $ACI_SUBNET_NAME \
+    --address-prefix $ACI_SUBNET_RANGE
+```
+
```

Comment:
can you also add the command to enable ACI in the subscription here?
```cli
az provider register -n Microsoft.ContainerInstance
```

Reply:
I have that in the instructions above. Do you think we need to add it again here?

Reply:
ok, it's fine then.